DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -702,7 +702,8 @@ internal async Task OnReadyEventAsync(ReadyPayload ready, JArray rawGuilds, JArr
             this,
             new()
             {
-                ShardId = shardId
+                ShardId = shardId,
+                GuildIds = [.. guilds.Select(guild => guild.Id)]

```

Comment:
Github diffs gaslighting me into almost spewing about how this will continue to increase guilds in each payload, smh